### PR TITLE
Fix render_nanomaps action

### PR DESCRIPTION
**Pull request by Toby222**
## What Does This PR Do
Use up-to-date VM and updated SpacemanDMM, with source modifications retained.

## Why It's Good For The Game
Keeps maps up-to-date

## Testing
Ran action on my fork successfully.

## Changelog
:cl:
fix: Fixed nanomaps
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
The runner version bump makes sense, however I will ask, what's been changed in the binary?

**Comment by Toby222** [comment]
> The runner version bump makes sense, however I will ask, what's been changed in the binary?
I also added a link to the updated source.
The TL;DR is that I got warnings about BYOND being outdated when I was testing the action, and I figured there's no good reason not to catch up with a couple years' worth of updates to SpacemanDMM.

**Comment by AffectedArc07** [comment]
> > The runner version bump makes sense, however I will ask, what's been changed in the binary?
> 
> I also added a link to the updated source. The TL;DR is that I got warnings about BYOND being outdated when I was testing the old binary, and I figured there's no good reason not to catch up with a couple years' worth of updates to SpacemanDMM.

I am fine to have this merged if you revert the binary change. I have been meaning to just make a PR to sdmm itself to add a `nanomaps` mode that hides all objects as well as native resize options, its easier than us maintaining our own fork, plus then we can use the `_build_dependencies.sh` in the repo root.

**Comment by Toby222** [comment]
> I am fine to have this merged if you revert the binary change. I have been meaning to just make a PR to sdmm itself to add a `nanomaps` mode that hides all objects as well as native resize options, its easier than us maintaining our own fork, plus then we can use the `_build_dependencies.sh` in the repo root.

Reverted the binary change (and the other stuff relating to it). Seems to work despite the BYOND version error.  
Feel free to update ParaSpacemanDMM in your own time as a stop-gap, I suppose.  
I undersand that binaries from first-time-committers aren't really the paragon of trustworthiness :p

File: tools/github-actions/nanomap-renderer-invoker.sh [hunk 1]
```diff
@@ -8,7 +9,7 @@ tools/github-actions/nanomap-renderer minimap -w 2040 -h 2040 "./_maps/map_files
 cd "data/nanomaps"
 mv "cyberiad_nanomap_z1.png" "Cyberiad_nanomap_z1.png"
 mv "delta_nanomap_z1.png" "Delta_nanomap_z1.png"
-mv "MetaStation_nanomap_z1.png" "MetaStation_nanomap_z1.png"
+#mv "MetaStation_nanomap_z1.png" "MetaStation_nanomap_z1.png" # What's even the point of this?
```
Comment (AffectedArc07) [inline tools/github-actions/nanomap-renderer-invoker.sh:12]: Just standardisation really

**Review by AffectedArc07 — APPROVED** [review]

**Comment by AffectedArc07** [comment]
> Your PR has been put on hold as your GitHub account is not linked to a Paradise forums account.
> 
> Please link your account at [https://www.paradisestation.org/forum/custom/link_github_account.php](https://www.paradisestation.org/forum/custom/link_github_account.php)
> 
> *Headcoders may ignore this in specific circumstances (Minor fix PR, trusted contributor from another codebase, anything at their discretion)*

PR author linked account

**Pull request merged**